packages/tpetra/core/test/FECrsGraph/FECrsGraph_UnitTests.cpp

```diff
@@ -169,6 +175,17 @@ void generate_fem1d_graph(size_t numLocalNodes, RCP<const Comm<int> > comm , Gra
     pack.element2node[i][0] = pack.uniqueMap->getGlobalElement(i);
     pack.element2node[i][1] = pack.uniqueMap->getGlobalElement(i) + 1;
   }
+
+  // Kokkos version of the element2node array
+  Kokkos::resize(pack.k_element2node,numLocalElements);
+  auto k_e2n = pack.k_element2node;
+  auto l_umap = pack.uniqueMap->getLocalMap();
+  Kokkos::parallel_for(numLocalElements, KOKKOS_LAMBDA(const size_t i) {
```

Comment:
@csiefer2 I think this is (part of) the problem -- you're using the default execution space here.  In a CUDA-enabled build, the default execution space is Cuda, even if Node=Serial.